users/user-manager.js

```diff
@@ -0,0 +1,74 @@
+var response = require('../Objects/response.js');
+var userModels = require('./user-models.js');
+var Student = userModels.Student;
+var Admin = userModels.Admin;
+
+var createUser = (reqData, createCallback) => {
+  if (reqData.role == 'Student') {
+    // Create new student.
+    var student = new Student({
+      approvalStatus: false,
+      email: reqData.email,
+      firstName: reqData.firstName,
+      lastName: reqData.lastName,
+      points: 0,
+      role: 'Student',
+      studentId: 0
+    });
+  // TODO(ryanfaulkenberry100): Write student to database.
+
+  } else if (reqData.role == 'Admin') {
+    // Create new admin.
+    var admin = new Admin({
+      email: reqData.email,
+      firstName: reqData.firstName,
+      lastName: reqData.lastName,
+      role: 'Admin'
+    });
+
+  // TODO(ryanfaulkenberry100): Write admin to database.
```

Comment:
Check indentation.

If the same variable name is used for the new Admin/Student, you can just call ```save()``` once (after the if-else block), in which case you'd only have one TODO for saving users.  See ```report-manager.js```.